float64 x1
float64 x2
float64 y
---
bool success